cclang/CMakeLists.txt

```diff
@@ -111,7 +111,7 @@ endif (WIN32)
 
 # set that name of the main output file as a target name
 if (NOT DEFINED COMMON_CLANG_LIBRARY_NAME)
-    message(FATAL_ERROR "COMMON_CLANG_LIBRARY_NAME not set!")
+    set(COMMON_CLANG_LIBRARY_NAME common_clang)
```

Comment:
IIRC, there is similar issue in the root CMakeLists.txt. Could we remove them?

Reply:
I think it should stay there. When some client includes opencl-clang from root, he won't be able to pass this flag to cclang with current root CmakeLists.txt